AmazonChimeSDK/AmazonChimeSDK.xcodeproj/project.pbxproj

```diff
@@ -347,6 +346,7 @@
 				5A15821E23C4020D0099F8FA /* Products */,
 				5A1582A023C403120099F8FA /* Frameworks */,
 				B2624DE0EEE7B7B1BCEB6407 /* Generated Mocks */,
+				C6C2F104250BD96F001678B8 /* Recovered References */,
```

Comment:
What is Recovered References?

Reply:
Good catch! According to: https://stackoverflow.com/a/46465477, Xcode tried to keep the reference and move it to Recovered References if the file is deleted. I wasn't able to find the original file in Xcode so just went ahead and deleted it from the file system. Will remove these as well.